package/MDAnalysis/coordinates/chain.py

```diff
@@ -155,8 +242,84 @@ def __init__(self, filenames, **kwargs):
         #: source for trajectories frame (fakes trajectory)
         self.__chained_trajectories_iter = None
 
+        # calculate new start_frames to have a time continuous trajectory.
+        if continuous:
+            filetypes = np.unique([r.format for r in self.readers])
+            if not len(filetypes) == 1:
+                raise RuntimeError("Continuous only supported with all files "
```

Comment:
I would probably make it `ValueError` because the input arguments do not have the correct value – they should contain the same kind of files.